src/verifier.js

```diff
@@ -78,7 +78,7 @@ class LocalVerifier {
         const payload = { [`${this.options.entity}Id`]: id };
         done(null, entity, payload);
       })
-      .catch(error => error ? done(error) : done(null, error));
+      .catch(error => error ? done(error) : done(null, error, { message: 'Invalid login' }));
```

Comment:
Maybe instead of this we should actually just reject with an error instead of returning `false` on line https://github.com/feathersjs/feathers-authentication-local/blob/master/src/verifier.js#L58. What do you think?

I was initially following how passport expects results to come back but I think in reality we do want to reject with either a 404, 400, or 403 error if invalid credentials are passed in. Likely a 404 since it is during the query look up by username that we can't find a user.